### PR TITLE
Add documentation

### LINKED ISSUE #379: Add self-hosting instructions and general documentation
There is no `example.env` file to quickly edit config.
There's also no simple way of registering the slash commands.

### PR DESCRIPTION
**Pull request by Ta1al**
Added `example.env` from required info `./src/config.ts`

Fixes #379

### CODE REVIEW

**Comment by Ta1al** [comment]
@promise You may want to add comments to this file (or you should).

**Comment by promise** [comment]
> @promise You may want to add comments to this file (or you should).

I will probably only add the env variables that make sense for someone to configure when they self-host, so probably remove access variables at least.

I'll take a look 👍

File: README.md [hunk 1]
```diff
@@ -13,6 +13,50 @@ Countr is an advanced counting bot which can manage a counting channel in your g
 
 Read more about Countr on our [website](https://countr.xyz)
 
+## Setup for private use
```
Comment (aditya20-b) [inline README.md]: Setup for self hosting would be better wording tbh

**Review by Ta1al — COMMENTED** [review]
lgtm

**Review by aditya20-b — COMMENTED** [review]
Aside from https://github.com/countr/countr/pull/380/files#r945494772 lgtm

**Review by aditya20-b — APPROVED** [review]

**Pull request merged**